src/syft/grid/duet/__init__.py

```diff
@@ -295,6 +224,7 @@ def join_duet(
     target_id: str = "",
     network_url: str = "",
     loopback: bool = False,
+    exchangeIds_method: Callable = manual_exchangeIds.client_get_then_send,
```

Comment:
Same here, change to 🐍 `snake_case` for variables.
The place for 🐫 `CamelCase` would be for `ClassNames`.

Reply:
resolved in a new commit